### PR TITLE
home page

### PR DESCRIPTION
**Pull request by sumitk-singh**
### codes to be review in the files-
- Navbar.jsx
- ProductsNavigator.jsx
- OfferBanner.jsx
- PriceCard.jsx
- Footer.jsx

### CODE REVIEW

File: src/components/Footer/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import "../../pages-css/home-page-css/page-layout-handeler.css";
```
Comment (iRohitGaur) [inline src/components/Footer/Footer.jsx]: suggestion: spelling of `handler`

**Review by iRohitGaur — APPROVED** [review]
Good to merge 👍🏻

**Review by srishtigautam21 — APPROVED** [review]
The code looks good to merge

File: src/components/Footer/Footer.jsx (hunk 1 above)
Comment (sumitk-singh) [reply src/components/Footer/Footer.jsx]: okay

**Pull request merged**